ui/components/src/TimeChart/TimeChart.tsx

```diff
@@ -257,8 +256,6 @@ export const TimeChart = forwardRef<ChartInstance, TimeChartProps>(function Time
     }
 
     return option;
-    // tooltipPinnedCoords is needed in dep array so crosshair stays beside pinned tooltip onClick
-    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [
     data,
     seriesMapping,
```

Comment:
Should this be `updatedSeriesMapping`?

Reply:
So `updatedSeriesMapping` gets populated inside of this useMemo (and set in option.series) whenever `pinnedCrosshair` state changes

Reply:
Ah sorry about my overaggressive pinging about that. Github's partial file showing bit me on missing some of the extra context.